src/StreamingDataFrames/.gitignore

```diff
@@ -0,0 +1,25 @@
+#Ignore PyCharm files
+.idea/
+
+# Jupyter checkpoints
+.ipynb_checkpoints/
+
+# pytest cache
+.pytest_cache
+
+# coverage reports
+.coverage
+
+#ignore environment
+env/
```

Comment:
might be a good idea to add `venv/` too since that's what pycharm uses by default

Reply:
Yep, I can include it to one of the next PRs